cli/pcluster/commands.py

```diff
@@ -34,12 +34,14 @@
 
 LOGGER = logging.getLogger(__name__)
 
+CW_DASHBOARD_CFN_PATH = os.path.realpath(
+    os.path.join(os.path.dirname(__file__), "cw-dashboard-substack.cfn.yaml")
+)  # FIXME
```

Comment:
Let's remove the fixme and make the code ready to be merged.
We can consider adding an additional configuration parameter like the hit_template_url one.

Reply:
done, changed the path to ./../cloudformation/cw-dashboard-substack.cfn.yaml to put the cfn yaml file with the others

Reply:
Will change back to #FIXME as it should be a remote file read, as you said in one of your later comments

Reply:
Additional configuration parameter added, FIXME removed